.gitignore

```diff
@@ -18,6 +18,7 @@ app.db
 
 # Static files at 'app/static/vendor'
 app/static/vendor
+app/static/admin
```

Comment:
Because git was showing this under untracked files. I feel this is generated under installation process and not need to present under git repo.